docs/guides/timestamp-pruning.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Timestamp pruning
+sidebar_position: 4
+---
+
+Quickwit indexes documents in bacthing fashion by outputing self contained index files called [splits](https://quickwit.io/docs/overview/architecture/#splits). Overtime, the number of splits created in a Quickwit index grows as long as new documents are being indexed. This number of splits linearly hurts query performance because Quickwit needs to examine all the splits in an index during query execution.
```

Comment:
```suggestion
Quickwit indexes documents in batching fashion by outputting self-contained index files called [splits](../architecture/#splits). Over time, the number of splits created in a Quickwit index grows as long as new documents are being indexed. This number of splits linearly hurts query performance because Quickwit needs to examine all the splits in an index during query execution.
```

we need to use relative path for the docs